### PR TITLE
Add docs for /apps for Canvas Public Beta

### PR DESCRIPTION
**Pull request by jessicalc**
@jgaul @mauerbac @lucasoptimizely @jprovine @tscanlin 

Hi, these are the docs that will sit at http://developers.optimizely.com/apps. Original docs are here: https://docs.google.com/document/d/1y2WI0rDzLMKdefF35u-ej6cJzmul7Abyvhep0xwFpTc/edit#

Changes: 
- Remove content for /canvas
- Make /canvas redirect to /apps with Javascript (not 301 since this was never indexed). Let me know if there's a better way to do this.
- Update header partial to exclude/hide apps for now. Also tweaked the logic and added Integrations to it.
- Update config.js

In a later diff I will add Apps to the sidebar nav and the header nav. They will remain hidden until the beta launch.

### CODE REVIEW

**Comment by jessicalc** [comment]
Note that the Starter Kits and SDKs lists aren't final - they depend on third-party factors. Will update when finalized ASAP.

File: src/content/canvas/index.md [hunk 1]
```diff
@@ -1,180 +1,8 @@
 ---
```
Comment (tscanlin) [inline src/content/canvas/index.md]: Can you do redirects in a similar fashion to the way other redirects are done in this repo, see an example here: https://github.com/optimizely/developers.optimizely.com/blob/master/src/files/javascript/guide/index.html#L6

File: src/content/apps/index.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+**Welcome! This page walks you through everything you need to know so you can build apps using Optimizely.**
+
+At Optimizely, we’re working hard to expand our developer platform to empower customers and developers to extend and integrate with Optimizely in new ways.
+
+In addition to our APIs, we’ve also built the Optimizely Canvas<sup>beta</sup> framework, which allows developers to embed custom experiences inside the native Optimizely interface.
+
+We’re excited for customers and developers to leverage these powerful tools to build apps that extend the Optimizely experience.
+
+## What is Optimizely Canvas <sup>beta</sup>?
```
Comment (tscanlin) [inline src/content/apps/index.md]: You may want to write this with HTML so it doesn't have the beta label in the headings ID. Eg:
`<h2 id="what-is-optimizely-canvas">What is Optimizely Canvas <sup>beta</sup></h2>`

Comment (tscanlin) [inline src/content/apps/index.md]: same for above: "# Apps<sup>beta</sup> Developer Guide"

File: src/partials/_header.html [hunk 3]
```diff
@@ -31,11 +31,16 @@
             </a>
             <ul class="unstyled nav nav--stacked dropdown background--super-dark soft--sides collapsible--small is-collapsed" data-auto-close>
               {% for key, item in paths.navigation %}
-                {% if item.path !== 'samples' && item.path !== 'overview' && item.path !== 'integrations' && item.path !== 'canvas' %}
+                {% if item.path === 'rest' || item.path === 'javascript' || item.path === 'ios' || item.path === 'android' %}
                   <li class="push-half">
                     <a href="/{{ item.path }}/introduction/">{{ item.title }}</a>
                   </li>
                 {% endif %}
+                {% if item.path == 'integrations' %}
```
Comment (tscanlin) [inline src/partials/_header.html]: what is this supposed to do and why just for integrations?

**Comment by tscanlin** [comment]
this looks good to me, just a few minor comments.

File: src/partials/_header.html (hunk 3 above)
Comment (jessicalc) [inline src/partials/_header.html]: This adds integrations to the Header nav (it was previously missing, but displaying in the sidebar, so I thought it should be added in). When we're ready to make the Apps content public, we'll add this to the nav as well. Does that makes sense?

File: src/content/apps/index.md (hunk 2 above)
Comment (jessicalc) [inline src/content/apps/index.md]: Thank you!! I was wondering how to do this.

File: src/partials/_header.html (hunk 3 above)
Comment (tscanlin) [inline src/partials/_header.html]: yep, that makes sense, thanks for clarifying!

File: src/content/apps/index.md (hunk 2 above)
Comment (jessicalc) [inline src/content/apps/index.md]: Oh no, it doesn't seem to have done anything - the title string doesn't port over to the nav with <superscript>. I don't think this is ship-stopping, so I'll leave as is

**Comment by jessicalc** [comment]
@vraja2

**Pull request merged**